meta/runtime.yml

```diff
@@ -155,6 +156,13 @@ plugin_routing:
     execute_lambda:
       # Deprecation for this alias should not *start* prior to 2024-09-01
       redirect: amazon.aws.lambda_execute
+    rds_param_group:
+      redirect: amazon.aws.rds_instance_param_group
+      deprecation:
+         removal_version: 9.0.0
```

Comment:
We're going to remove this alias after just 6 months?

We have a mass of aliases at the minute, do we want to drop them all at the same time, maybe we drop all of them with 10.0.0 (since it's a nice round number) and it would give us 1 year from announcing the deprecation to performing it.

Reply:
Right! I thought 9.0 was further away, but apparently I was wrong. Updated to 10.0.0. Thanks-